doc-content/enterprise-only/release-notes/rn-deprecated-issues-ref.adoc

```diff
@@ -1,16 +1,19 @@
 [id='rn-deprecated-issues-ref']
-= Deprecated components 
+= Deprecated components
 
-The components listed in this section have been deprecated.
+The components listed in this section have been deprecated or removed.
 
 
 //== Installation
 
 ifdef::PAM[]
-== Legacy Test Scenarios tool
+== Legacy process designer
```

Comment:
Although this information is technically correct, I'm wondering about the wording. "start migrating your processes to the new designer", "Create all new processes in the new process designer."

It may be the accepted way how these disclaimers are written, in which case please disregard my comment, but I do consider this wording very direct, brief, commanding.

Reply:
Well, we do mitigate with the caveat 'If you intend to use the new process designer..." And where possible we try to use the imperative (Start migrating... Create... which can seem as you say.

Reply:
Do as you will, language is not my forte.